ext/phar/phar_object.stub.php

```diff
@@ -134,7 +134,7 @@ public function setMetadata(mixed $metadata) {}
     public function setSignatureAlgorithm(int $algo, ?string $privateKey = null) {}
 
     /**
-     * @param resource $stub
+     * @param resource|string $stub
      * @return bool
      */
     public function setStub($stub, int $length = -1) {}
```

Comment:
This change itself looks good, but I think there's a problem with the second parameter as well: when `$stub` is a `string`, a 2nd argument is not accepted at all.

Reply:
(this means that the default value is not correct, we usually mark such cases as `UNKNOWN`, therefore the signature becomes overloaded)